scripts/build_web.py

```diff
@@ -0,0 +1,37 @@
+from shutil import copyfile
+from subprocess import check_output
+import sys
+import os
+import platform
+import subprocess
+
+Import("env")
+
+
+def is_tool(name):
+    cmd = "where" if platform.system() == "Windows" else "which"
+    try:
+        check_output([cmd, name])
+        return True
+    except:
+        return False;
+
+
+def build_web():
+    if is_tool("npm"):
```

Comment:
-1 for compiled artifact. Because I do not like redundant data.

And then this should fail on on error.